Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace BootCamp.Chapter1
+﻿//using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011611)